bower.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "stencil-select",
+  "version": "0.1.0",
+  "description": "A styleable, accessible <select> component.",
+  "authors": [
+    "Mobify"
+  ],
+  "keywords": [
+    "mobify",
+    "components",
+    "patterns"
+  ],
+  "main": "",
+  "dependencies": {
+    "spline": "1.0.1"
+  },
+  "devDependencies": {
+    "vellum": "~1.0.0",
+    "modularized-normalize-scss": "~3.0.1"
+  },
```

Comment:
We could have a core stencil dependency – probably a co

I’m a little confused as to why we’d test with Vellum and Normalize in place if they aren’t dependencies… the tests should be as close to unit tests as possible, so if those styles are not “required” then we shouldn’t be applying them in the test environment. That was what the fixture css was for in Stencil 1.x.

Also if Vellum and Normalize aren’t dependencies, we shouldn’t be `@import`-ing them in the component stylesheet. However, then the components can’t rely on Vellum variables (unless we just depend on `vellum/_variables.scss`).